Sources/JobsPostgres/PostgresJobsQueue.swift

```diff
@@ -303,8 +305,8 @@ public final class PostgresJobQueue: JobQueueDriver {
 
             let jobs = try await getJobs(withStatus: status)
             self.logger.info("Moving \(jobs.count) jobs with status: \(status) to job queue")
-            for jobId in jobs {
-                try await self.addToQueue(jobId: jobId, connection: connection)
+            for (jobId, delayUntil) in jobs {
+                try await self.addToQueue(jobId: jobId, connection: connection, delayUntil: delayUntil)
```

Comment:
You can only get here if a job is failed or processing. If they are in this state then they will have reached their delayUntil time, so there is no need to set this anymore. Because of this then there is no need to store `delayUntil` in the jobs table, as it isn't used anywhere else.

Reply:
Understood!